package.json

```diff
@@ -2,21 +2,21 @@
   "name": "metadecidim",
   "private": true,
   "dependencies": {
-    "@decidim/browserslist-config": "^0.28.1",
-    "@decidim/core": "^0.28.1",
+    "@decidim/browserslist-config": "~0.29.0-rc2",
+    "@decidim/core": "~0.29.0-rc2",
     "@decidim/elections": "^0.28.1",
```

Comment:
Why didn't you update the `@decidim/elections` package?

Reply:
Nice catch, I missed this one! 

We (@decidim/maintainers) actually dropped official support of decidim-elections in this version. In the next few months @decidim/product will do a blog post explaining in detail what happened. Just as a summary it was a development that isn't maintained anymore, so it's safe to drop it.

Reply:
I removed it on [a8d9860](https://github.com/decidim/metadecidim/pull/129/commits/a8d986070e4c3cc36ef4072a9d7572e14a14ff6f)